uni/lib/model/providers/startup/session_provider.dart

```diff
@@ -46,8 +50,16 @@ class SessionProvider extends StateProviderNotifier<Session> {
     return state!;
   }
 
+  static Future<void> _invoke(Uri uri) async {
+    if (await canLaunchUrl(uri)) {
+      await launchUrl(uri, mode: LaunchMode.inAppBrowserView);
+    } else {
+      Logger().e('Could not launch $uri');
+    }
+  }
+
   Future<void> postAuthentication(
-    BuildContext context,
+    material.BuildContext context,
```

Comment:
Why is this needed?

Reply:
I am not sure what class was having conflicts with a new import, but there was one.

Reply:
It is the `Flow`